include/fs/fs.h

```diff
@@ -160,6 +161,17 @@ struct fs_file_system_t {
 					struct fs_statvfs *stat);
 };
 
+#define FO_READ		0x01
```

Comment:
> align with values below

I may imagine that's just a rendering artifact due to use of tabs. @de-nordic, can you confirm? One obvious way to avoid this confusion is to avoid use of tabs. Zephyr mandates use of tabs only for indentation (i.e., at the beginning of line), and the less we use tabs beyond that, the better.

Reply:
OK. Will fix.